modules/op-release-notes-1-11.adoc

```diff
@@ -202,4 +202,40 @@ With this command, you can also specify a custom GitHub API URL by using the `--
\ No newline at end of file
+* This update fixes the `tkn pr delete` command for correctly using the `keep` flag. Now, if the value of the `keep` flag equals the number of associated task runs or pipeline runs, the `tkn pr delete` command returns exit code `0` along with a message.
+
+[id="release-notes-1-11-1_{context}"]
+== Release notes for {pipelines-title} General Availability 1.11.1
+
+With this update, {pipelines-title} General Availability (GA) 1.11.1 is available on {product-title} 4.12 and later versions.
+
+[id="fixed-issues-1-11-1_{context}"]
+=== Fixed issues
+
+* Before this update, a task run could fail, displaying a mount path error, when a running or pending pod was preempted. With this update, a task run does not fail when the cluster causes a pod to be destroyed and re-created.
+
+* Before this update, a shell script in a task had to be run as root. After this update, the shell scripts image has the non-root user ID set, so a task that includes a shell script, such as the `git-clone` task, can be run as a non-root user within the pod.
+
+* Before this update, in {pipelines-title} 1.11.0, when a pipeline run was included in a Tekton bundle, the `kind` value was sometimes wrongly set to `Task`. The issue did not exist in previous versions of {pipelines-title}. After this update, the `kind` value is set correctly.
+
+* Before this update, the `disable-ha` flag was not correctly passed to the `tekton-pipelines` controller, so the High Availability (HA) features of {pipelines-title} could not be enabled. After this update, the `disable-ha` flag  is correctly passed and you can enable HA features as required.
+
+* Before this update, you could not set the URL for {tekton-hub} and {artifact-hub} for the hub resolver, so you could use only the preset addresses of {tekton-hub} and Partifact-hub}. After this update, you can configure the URL for {tekton-hub} and {artifact-hub} for the hub resolver, for example, to use a custom {tekton-hub} instance that you installed.
+
+* After this update, the SHA digest of the `git-init` image corresponds to version 1.10.5, which is the latest released version of the image.
+
+* Before this update, the `tekton-pipelines-controller` leader election used a config map named `config-leader-election`. This name is the default value for knative controllers, so the configuration process for {pipelines-shortname} could affect other controllers and vice versa. After this update, the leader election uses a unique config name, so the configuration process for {pipelines-shortname} does not affect other controllers and is not affected by other controllers.
+
+* Before this update, when a user who has no write access to a GitHub repository opened a pull request, {pac} CI/CD actions would show as skipped in GitHub. After this update, {pac} CI/CD actions are executed correctly in this case.
```

Comment:
> After this update, {pac} CI/CD actions are executed correctly in this case.

How about adding what is shows after the update 
I mean before update it use to show `skipped` but after update it shows `Pending approval`

Reply:
Fixed, thanks @shipsing